src/components/graph/search/GraphSearch.tsx

```diff
@@ -164,23 +151,23 @@ function GraphSearch(props: Props) {
 	);
 
 	const onModeSelect = (newMode: GraphSearchMode) => {
+		setTimestamp(null);
```

Comment:
why you're resetting the timestamp?

Reply:
That's made to prevent search happening when mode changes from timestamp to history by switcher rather than by submit.